mentorship.html

```diff
@@ -6,7 +6,52 @@
 <div class="page-header">
     <h1>Mentorship Programme</h1>
 </div>
-<div class="container">
-    TODO
-</div>
- <br> <br><br> <br><br> <br>
\ No newline at end of file
+<div class="container home">
+    <div class="intro">
+        <div class="row justify-content-center">
+            <div class="col-12 col-md-6 text-center py-6 overview">
+                <p class="lead">Mentorship is a fantastic way to get support for your personal growth and career development. Mentorship relationships can be rewarding to both people, personally and professionally. It's an opportunity to develop communication skills, expand your viewpoints, and consider new ways of approaching situations. And both partners can advance their careers in the process.</p>
+            </div>
+        </div>
+        <div class="row registration justify-content-center">
+                <div class="col-xs-12 col-md-5 mentor">
+                    <div class="row">
+                        <p>You should apply to be a <span>mentor</span> if you:</p>
+                        <ul>
+                            <li>1. Want to extend your professional network</li>
+                            <li>2. Want to contribute to the community</li>
+                            <li>3. You are ready to share expertise</li>
+                            <li>4. You want to get a new perspective and learn from your mentees</li>
+                        </ul>
+                    </div>
+                    <div class="row">
+                        <p class="mt-4 d-flex justify-content-bottom">
+                            <button class="btn btn-primary" data-tf-popup="" data-tf-iframe-props="title=Mentor Registration" data-tf-medium="snippet">Join as a Mentor</button>
```

Comment:
here we have to add the form that Eleonora created and open in a new tab or in the same but as a modal windows. 
```suggestion
                            <button class="btn btn-primary">Join as a Mentor</button>
```

this html was for typeform and we are not using right now.

Reply:
Removed typeform related data.